src/olympia/activity/utils.py

```diff
@@ -297,6 +313,10 @@ def notify_about_activity_log(
             task_user = {get_task_user()}
         except UserProfile.DoesNotExist:
             task_user = set()
+        # Update the author and from_email to use the real name in all cases.
```

Comment:
Could you modify this comment to indicate why ? (i.e. because it's for the email sent to reviewers)

Also the comment a few lines below mentions building the context for reviewers but it's now built here so it needs to be adjusted as well.